app/views/devise/two_step_verification_session/new.html.erb

```diff
@@ -23,6 +23,15 @@
           autocomplete: 'off',
           'aria-describedby': 'code-hint' %>
       </div>
+      <div data-module="toggle" class="if-no-js-hide">
+        <%= link_to "Can’t get your verification code?", "#", class: "js-toggle js-toggle-target" %>
+        <div class="js-toggle-target if-js-hide">
+          <h4>Contact <%= link_to "GOV.UK", "https://www.gov.uk/" %></h4>
```

Comment:
I'm not sure this should link, I think that's not a helpful link in this context.